### DISCUSSION TITLE
Best way to delete stipe customer when an auth user has been deleted?

### DISCUSSION THREAD

**Discussion by bennik88** [supabase/supabase#16023] [Questions]
I want to delete my stripe customer when an auth user is being deleted. Is it a good idea to call a `pg_net` async request with the API KEY inside of the trigger function, to delete the stripe user? It feels kind of like something you would generally accomplish in the application layer. But as I can't track when an auth user is been deleted from the dashboard I think of creating a trigger.

**Comment by bennik88** [comment]
I know used webhooks: [Supabase Webhook guides](https://supabase.com/docs/guides/database/webhooks)

**Comment by ryanburry** [comment]
Did you edit the stripe user's table to change the FK relation to "Cascade"? That will remove the user from the table once the user is removed from auth.

In the Supabase dashboard, go to stripe user's table -> Edit the FK id column -> in the Foreign Key Relation part, click Edit relation -> Action if referenced row is removed: Cascade
